sentry/map.jinja

```diff
@@ -0,0 +1,27 @@
+{%- load_yaml as defaults %}
+Debian:
+  pkgs:
```

Comment:
As you wish. I just wanted to make it obvious in map.jinja that support may be added by preparing the structure.